### PR TITLE
Fixed broken Python badge in README.md

### PR DESCRIPTION
**Pull request by stevenhastings**
Removed broken Python badge and replaced it with a working badge.

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix
- [ ] New feature
- [ ] This change requires a documentation update

## Checklist:

- [ ] My code follows PEP8 style guide
- [ ] I have removed unnecessary print statements from my code
- [ ] I have made corresponding changes to the documentation if necessary
- [ ] My changes generate no errors
- [ ] No commented-out code
- [ ] Size of pull request kept to a minimum
- [ ] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ Our research into this project stretches far and wide, covering a multitude of l
 
 # Tech Stack
 ![MongoDB](https://img.shields.io/badge/MongoDB-%234ea94b.svg?style=for-the-badge&logo=mongodb&logoColor=white)
-![Python](https://img.shields.♠io/badge/python-3670A0?style=for-the-badge&logo=python&logoColor=ffdd54)
+![Python](https://img.shields.io/badge/Python-14354C?style=for-the-badge&logo=python&logoColor=white)
```
Comment (MelodyLayne) [inline README.md:45]: This change looks fine, but also found that just removing the ♠ icon from the url resulted in a working python image. The two images are different in their color schemes , though that may not matter.

**Review by MelodyLayne — APPROVED** [review]
The change looks acceptable to me. However I think the parameters of the pull request normally include completing the checklists associated with the PR. For this small of an issue it probably doesn't matter, but for future and larger PR's you'll likely be asked to complete that along with a short loom video detailing the changes. A loom here would be slight overkill though so I would have left it out too.

**Review by jettyblue — APPROVED** [review]
Nice quick fix on this bug. Good lookin out!

File: README.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ Our research into this project stretches far and wide, covering a multitude of l
 
 # Tech Stack
 ![MongoDB](https://img.shields.io/badge/MongoDB-%234ea94b.svg?style=for-the-badge&logo=mongodb&logoColor=white)
-![Python](https://img.shields.♠io/badge/python-3670A0?style=for-the-badge&logo=python&logoColor=ffdd54)
```
Comment (jack-semder) [inline README.md:45]: Excellent change, this looks great!

**Review by jack-semder — APPROVED** [review]

**Review by brimjustin — COMMENTED** [review]
Great catch and change on something that was noticeable!

**Review by L0lbrich — APPROVED** [review]
easy win!

**Pull request merged**